easypost_java_style.xml

```diff
@@ -1,7 +1,4 @@
 <?xml version="1.0"?>
-<!DOCTYPE module PUBLIC
-          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
-          "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
Why remove the doctype declaration?

Reply:
My mistake, my IDE was complaining so I removed it. Figure out how to resolve the issue and it's been readded.